rest/route/service.go

```diff
@@ -265,6 +265,7 @@ func AttachHandler(app *gimlet.APIApp, opts HandlerOpts) {
 	app.AddRoute("/versions/{version_id}/builds").Version(2).Get().Wrap(requireUser, viewTasks).RouteHandler(makeGetVersionBuilds(env))
 	app.AddRoute("/versions/{version_id}/restart").Version(2).Post().Wrap(requireUser, editTasks).RouteHandler(makeRestartVersion())
 	app.AddRoute("/versions/{version_id}/annotations").Version(2).Get().Wrap(requireUser, viewAnnotations).RouteHandler(makeFetchAnnotationsByVersion())
+	app.AddRoute("/tasks/{task_id}/revoke_github_dynamic_access_tokens").Version(2).Delete().Wrap(viewTasks).RouteHandler(makeRevokeGitHubDynamicAccessTokens())
```

Comment:
also nit: consider calling `/tasks/{task_id}/github_dynamic_access_tokens` since the agent route equivalent is called that. The `DELETE` part of the route signature should be clear enough that it's invalidating the token.